lib/membrane_ice_plugin/candidate_port_assigner.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Membrane.ICE.CandidatePortAssigner do
+  @moduledoc false
+
+  @min_port 40000
+  @max_port 65535
+
+  @spec assign_candidate_port() :: {:ok, number()} | {:error, :no_free_candidate_port}
+  def assign_candidate_port() do
+    if Process.whereis(__MODULE__) == nil,
+      do: Registry.start_link(keys: :unique, name: __MODULE__)
```

Comment:
Maybe we can do this in application module https://hexdocs.pm/elixir/master/Application.html